src/ScheduledTasks/ScheduledTasks.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <PackageDescription>Durable Task Scheduled Tasks Client</PackageDescription>
+    <EnableStyleCop>true</EnableStyleCop>
+    <VersionSuffix>preview.1</VersionSuffix>
```

Comment:
Do we need any pipeline updates to build the signed nugets for this new project?

Reply:
will take a look

Reply:
checked they are included in sln, thanks